build-system/externs/amp.extern.js

```diff
@@ -223,7 +241,18 @@ window.__AMP_TOP;
 window.__AMP_PARENT;
 window.__AMP_WEAKREF_ID;
 window.__AMP_URL_CACHE;
-window.AMP = {};
+window.__AMP_LOG;
+
+/** @type {boolean} */
+window.ENABLE_LOG;
+
+/** @type {../../src/mode.ModeDef} */
+window.__AMP_MODE;
+
+/** @type {boolean|undefined} */
+window.AMP_DEV_MODE;
+
+window.AMP;
```

Comment:
Is there a type definition for `window.AMP`?

Reply:
Not yet!